lib/mime.ex

```diff
@@ -164,6 +164,13 @@ defmodule MIME do
     unquote(Macro.escape(custom_types))
   end
 
+  @doc """
+  Returns a list of all known types, including custom types compiled into the MIME module.
+  """
+  def known_types do
```

Comment:
```suggestion
  @doc since: "2.1.0"
  @spec known_types() :: [String.t(), ...]
  def known_types do
```

Reply:
Applied a custom version of this suggestion since the type spec changed a bit :)